Apps/W1/HybridGP/app/src/Migration/Support/HelperFunctions.codeunit.al

```diff
@@ -2243,6 +2278,139 @@ codeunit 4037 "Helper Functions"
             Dimension.DeleteAll(true);
     end;
 
+    internal procedure CreateDimSet(ACTNUMBR_1: Code[20]; ACTNUMBR_2: Code[20]; ACTNUMBR_3: Code[20]; ACTNUMBR_4: Code[20]; ACTNUMBR_5: Code[20]; ACTNUMBR_6: Code[20]; ACTNUMBR_7: Code[20]; ACTNUMBR_8: Code[20]): Integer
+    var
+        TempDimensionSetEntry: Record "Dimension Set Entry" temporary;
+        DimensionValue: Record "Dimension Value";
+        GPSegments: Record "GP Segments";
+        DimensionManagement: Codeunit DimensionManagement;
+        NewDimSetID: Integer;
+    begin
+        if GPSegments.FindSet() then
```

Comment:
I prefer if not GpSegments.FindSet() then
               exit;

It has less nesting and is easier to read